api/src/main/java/org/openmrs/module/aijar/api/impl/AijarServiceImpl.java

```diff
@@ -302,4 +302,35 @@ private String replaceLettersWithNumber(String letter) {
 		}
 		return numberToReturn;
 	}
+	/**
+	 * @see org.openmrs.module.aijar.api.AijarService#stopActiveOutPatientVisits()
+	 */
+	public void stopActiveOutPatientVisits() {
+
+		SimpleDateFormat formatter = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
+
+		SimpleDateFormat formatterExt = new SimpleDateFormat("yyyy-MM-dd");
+
+		String formattedDate = formatterExt.format(new Date()) + " " + "00:00:00";
+
+		AdministrationService administrationService = Context.getAdministrationService();
+
+		String visitTypeUUID =administrationService.getGlobalProperty("ugandaemr.autoCloseVisit.visitTypeUUID");
+
+		VisitService visitService = Context.getVisitService();
```

Comment:
@slubwama Why not autowire the VisitService

Reply:
When you auto wire this is what happens to the tests
`Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private org.openmrs.api.AdministrationService org.openmrs.module.aijar.api.impl.AijarServiceImpl.administrationService; nested exception is org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type [org.openmrs.api.AdministrationService] is defined: expected single matching bean but found 2: adminServiceTarget,adminService`